yasir-khan-microverse-trials-main/index.html

```diff
@@ -0,0 +1,258 @@
+        </div>
+        <div class="form-check">
+          <input
+            type="checkbox"
+            class="form-check-input"
+            name=""
+            id=""
+            value="datavisualization"
+          />
+          Data Visualization
+        </div>
+        <div class="form-check">
+          <input
+            type="checkbox"
+            class="form-check-input"
+            name=""
+            id=""
+            value="challenges"
+          />Challenges
+        </div>
+        <div class="form-check">
+          <input
+            type="checkbox"
+            class="form-check-input"
+            name=""
+            id=""
+            value="opensourcecommunity"
+          />
+          Open Source Community
+        </div>
+        <div class="form-check">
+          <input
+            type="checkbox"
+            class="form-check-input"
+            name=""
+            id=""
+            value="gitter"
+          />
+          Gitter help rooms
+        </div>
+        <div class="form-check">
+          <input
+            type="checkbox"
+            class="form-check-input"
+            name=""
+            id=""
+            value="videos"
+          />
+          Videos
+        </div>
+        <div class="form-check">
+          <input
+            type="checkbox"
+            class="form-check-input"
+            name=""
+            id=""
+            value="city"
+          />City Meetups
+        </div>
+        <div class="form-check">
+          <input
+            type="checkbox"
+            class="form-check-input"
+            name=""
+            id=""
+            value="Wiki"
+          />
+          Wiki
+        </div>
+        <div class="form-check">
+          <input
+            type="checkbox"
+            class="form-check-input"
+            name=""
+            id=""
+            value="Forum"
+          />
+          Forum
+        </div>
+        <div class="form-check">
+          <input
+            type="checkbox"
+            class="form-check-input"
+            name=""
+            id=""
+            value="courses"
+          />
+          Additional Courses
+        </div>
+
+        <br />
+        <label for="" class="form-label">Any comments or suggestions?</label>
+        <textarea
+          name=""
+          id=""
+          cols="30"
+          rows="10"
+          placeholder="Enter your comments here..."
+        ></textarea>
+        <br />
+        <button type="submit" id="submit" class="btn btn-success">
+          Submit
+        </button>
+      </form>
+    </div>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
In order to maintain your code clean, you might also wish to delete any commented code from your code base.